src/base/pagination/test/clickEvent.test.ts

```diff
@@ -0,0 +1,40 @@
+import { elementUpdated, expect, fixture } from "@open-wc/testing";
+import { BasePagination } from "../index";
+
+describe("BasePagination", () => {
+  describe("Click events", () => {
```

Comment:
unifying other components, description should be "clickEvent"
ref: 
https://github.com/kintone-labs/kintone-ui-component/blob/master/src/button/test/clickEvent.test.ts#L5
https://github.com/kintone-labs/kintone-ui-component/blob/master/src/mobile/button/test/clickEvent.test.ts#L5